MinecraftClient/Mapping/Material.cs

```diff
@@ -777,5 +791,126 @@ public enum Material
         ChiseledNetherBricks,
         CrackedNetherBricks,
         QuartzBricks,
+        Candle,
+        WhiteCandle,
+        OrangeCandle,
+        MagentaCandle,
```

Comment:
Please sort the lines alphabetically after adding new entries, makes it easier to find material names (users will need to look into this file for AutoCraft)